kotlin-build-plugins/build.gradle.kts

```diff
@@ -1,14 +1,22 @@
 plugins {
     kotlin("jvm") version "1.8.21"
+    kotlin("kapt") version "1.8.21"
     id("io.gitlab.arturbosch.detekt") version "1.23.0"
-    id "org.jlleitschuh.gradle.ktlint" version "7.1.0"
+    id("org.jlleitschuh.gradle.ktlint") version "7.1.0"
 }
 
 repositories {
     mavenCentral()
 }
 
+buildscript {
+    dependencies {
+        classpath("org.jlleitschuh.gradle:ktlint-gradle:7.1.0")
+    }
+}
+
 dependencies {
     detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.23.0")
-    classpath ("org.jlleitschuh.gradle:ktlint-gradle:7.1.0")
+    testImplementation(kotlin("test"))
```

Comment:
Just commenting for posterity - this is build cleanup, and a better way to handle this ktlint dependency.

Even though this draft doesn't put code in this module, this is a better way to do this.